Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
+
+RUN apt update
+
+RUN mkdir /my_market
+
+WORKDIR /my_market
+
+COPY ./src ./src
+COPY ./requirements.txt ./requirements.txt
+
+RUN python -m pip install --upgrade pip
+RUN pip install -r ./requirements.txt
+
+CMD ["python", "src/manage.py", "runserver", "0:8008"]
```

Comment:
Use EXPOSE to document the ports used by the container: Add an EXPOSE instruction to your Dockerfile to document that your container listens on port 8008.

EXPOSE 8008